elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -719,9 +707,33 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
                                 ((ReloadableX509ExtendedTrustManager) trustManager).reload();
                             }
                         })
+                .addOperation(new SimpleOperationDefinitionBuilder(ElytronDescriptionConstants.INIT, RESOURCE_RESOLVER)
+                        .setRuntimeOnly()
+                        .build(), init(TRUST_MANAGER_UTIL))
                 .build();
     }
 
+    private static OperationStepHandler init(ServiceUtil<?> MANAGER_UTIL) {
+        return new ElytronRuntimeOnlyHandler() {
+            @Override
+            protected void executeRuntimeStep(OperationContext context, ModelNode operation) throws OperationFailedException {
+                try {
+                    ServiceName serviceName = MANAGER_UTIL.serviceName(operation);
+                    ServiceController<?> serviceContainer = null;
+                    if(serviceName.getParent().getCanonicalName().equals("org.wildfly.security.key-manager")){
```

Comment:
This could use KEY_MANAGER_CAPABILITY instead of "org.wildfly.security.key-manager"